symbolic-il2cpp/src/usym.rs

```diff
@@ -395,25 +395,19 @@ impl<'a> UsymSymbols<'a> {
     }
 
     /// Returns a [`UsymSourceRecord`] at the given index.
-    pub fn resolve_record(&self, index: usize) -> Option<UsymSourceRecord> {
+    pub fn get_record(&self, index: usize) -> Option<UsymSourceRecord> {
```

Comment:
I'm bemused by all this change of heart 😄

Reply:
dang, i wasn't expecting somebody to catch this. i've been doing a lot of rebasing and i _may_ have forgotten the final naming scheme that i was using in my previous PR, so i was met with a very awkward surprise when i discovered that i had two `resolve_records` while sorting out this PR 😛 took me a bit of time to remember that this was called `get_record`

Reply:
i recalled this because i thought choosing `resolve` instead of `get` for retrieving something by an index was a rather odd choice, certainly in rust-land.  Decided not to be a PR nitpicker and didn't comment on it... but here we are :)
(i judge myself for this)